bigquerydatatransfer/quickstart.rb

```diff
@@ -0,0 +1,39 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START bigquerydatatransfer_quickstart]
+# Imports the Google Cloud client library
+require "google/cloud/bigquery/data_transfer"
+
+project_id = "YOUR_PROJECT_ID"  # TODO: Update to your project ID.
+
+# Instantiate a client
+BigQueryDataTransfer = Google::Cloud::Bigquery::DataTransfer # Alias the module
+data_transfer = BigQueryDataTransfer.new
+
+# Get the full path to your project.
+project_path = BigQueryDataTransfer::V1::DataTransferServiceClient.project_path(
```

Comment:
Uggghhhhh.... this literally just returns `"projects/#{project_id}"`

I HATE these project path helpers. Disgusting.

LGTM though ~ we're stuck with this for now